janus_server/Cargo.toml

```diff
@@ -63,5 +64,6 @@ serde_test = "1.0.137"
 tempfile = "3.3.0"
 testcontainers = "0.13.0"
 test_util = { path = "../test_util" }
+tokio = { version = "*", features = ["test-util"] }
```

Comment:
I really want to introduce a "test-util" feature to our crate and use it instead of the hacky `#[doc(hidden)]` solution we use now, except that I think a crate can't control its own features when "self-importing" (e.g. for tests). :-(

Still, with how we're breaking the crates out, maybe we could do something like `#[cfg(test)]` OR `#[cfg(feature = "test-util")]`.